packages/frontend/src/components/ElectionForm/Details/ElectionDetailsForm.tsx

```diff
@@ -102,6 +102,7 @@ export default function ElectionDetailsForm({editedElection, applyUpdate, errors
                     type="text"
                     error={errors.description !== ''}
                     value={editedElection.description}
+                    helperText={errors.description || "Supports **bold** and [link text](url) formatting"}
```

Comment:
Ah, so is helperText and FormHelpText pretty much equivalent? Both just messages outside of the box? I'll keep that in mind